### PR TITLE
Fix/mention notification copy

### PR DESCRIPTION
**Pull request by johnxnguyen**
## What's new in this PR?

This PR adds changes to remove the `@` symbol from mentions in notification body text.

### CODE REVIEW

File: Source/Notifications/Push notifications/Notification Types/LocalNotificationContentType.swift [hunk 1]
```diff
@@ -31,7 +31,13 @@ public enum LocalNotificationContentType : Equatable {
             return .ephemeral(isMention: message.textMessageData?.isMentioningSelf ?? false)
         }
         
-        if let messageData = message.textMessageData, let text = messageData.messageText , !text.isEmpty {
+        if let messageData = message.textMessageData, var text = messageData.messageText , !text.isEmpty {
```
Comment (typfel) [inline Source/Notifications/Push notifications/Notification Types/LocalNotificationContentType.swift]: I thought we should only remove the @ symbol from the localisation part of the notification. 

Previous:
@ Mention from Eric: @Michelle Roxas hello?

New:
Mention from Eric: @Michelle Roxas hello?

Comment (johnxnguyen) [reply Source/Notifications/Push notifications/Notification Types/LocalNotificationContentType.swift]: Hmm, confusing... the copy is already *Mention from Eric: ......*

Comment (typfel) [reply Source/Notifications/Push notifications/Notification Types/LocalNotificationContentType.swift]: It wasn't before.

Comment (typfel) [reply Source/Notifications/Push notifications/Notification Types/LocalNotificationContentType.swift]: I'd confirm this with design.

Comment (mikeger) [reply Source/Notifications/Push notifications/Notification Types/LocalNotificationContentType.swift]: I asked Jen, I think @typfel is right.

Comment (johnxnguyen) [reply Source/Notifications/Push notifications/Notification Types/LocalNotificationContentType.swift]: Ah ok, the copy must have been updated but not the ticket. I'll close this then.

**Pull request closed without merging**